### PR TITLE
Add Power BI docs

### PR DESCRIPTION
**Pull request by roekatz**
null

### CODE REVIEW

File: docs/cloud/integrations/bi/power-bi.mdx [hunk 1]
```diff
@@ -2,14 +2,41 @@
...
-    Click for details
-</Card>
\ No newline at end of file
+After you connect Power BI, Elementary will automatically and continuously extend the lineage to the report/dashboard level.
+This will provide you end-to-end data lineage to understand your downstream dependencies, called exposures.
+
+### Create & authorize a Service Principal
+
+Elementary makes use of a service principal secret to authenticate with the Power BI API on your behalf.
+Please follow the [official guide by Microsoft](https://learn.microsoft.com/en-us/power-bi/developer/embedded/embed-service-principal) for creating and authorizing a service principal for Power BI.
```
Comment (oravi) [inline docs/cloud/integrations/bi/power-bi.mdx:11]: Should we include a short and simplified version of it here? Our users will find it hard to read this guide

File: docs/cloud/integrations/bi/power-bi.mdx [hunk 2]
```diff
@@ -2,14 +2,41 @@
...
+### Allow Power BI's Admin API
+Go to Power BI's Admin portal -> Tenant Settings -> Admin API settings
+Enable the following features, and apply at least for the Service principal's security group (created on the last step):
+- `Service principals can access read-only admin APIs`
+- `Enhance admin APIs responses with detailed metadata`
+- `Enhance admin APIs responses with DAX and mashup expressions`
+
+Those features will allow Elementary to get all required info for computing the data lineage graph.
+
+### Connecting Looker to Elementary
```
Comment (oravi) [inline docs/cloud/integrations/bi/power-bi.mdx:26]: PowerBI and not Looker?

File: docs/cloud/integrations/bi/power-bi.mdx (hunk 1 above)
Comment (roekatz) [reply docs/cloud/integrations/bi/power-bi.mdx:11]: I don't know - a shorter version might be harder to follow cause it's less detailed. Simplified is a challenge - there are multiple steps there.
Maybe you can consider writing a unified PowerShell script that does all or most of the work (they're giving there snippets for some of the stages)

Comment (oravi) [reply docs/cloud/integrations/bi/power-bi.mdx:11]: ok not a blocker, let's release and improve as we go

**Review by oravi — APPROVED** [review]

**Pull request merged**